src/apps/chatter/room/index.js

```diff
@@ -50,7 +56,7 @@ export function Room ({ id }) {
         ]}
       />
       <h1>{name}</h1>
-      <ChatBox roomId={id} />
+      <ChatBox roomId={parseInt(id)} />
```

Comment:
You should generally pass a radix of 10 when calling parseInt, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description